detection_rules/kbwrap.py

```diff
@@ -71,6 +72,36 @@ def upload_rule(ctx, rules, replace_id):
     return results
 
 
+@kibana_group.command('import-rules')
+@click.argument('rules', nargs=-1, required=True)
+@click.option('--overwrite', '-o', is_flag=True, help='Overwrite existing rules')
+@click.option('--overwrite-exceptions', '-e', is_flag=True, help='Overwrite exceptions in existing rules')
+@click.option('--overwrite-action-connectors', '-a', is_flag=True,
+              help='Overwrite action connectors in existing rules')
```

Comment:
Torn between whether to decouple rule updates from list updates. Imagine users wanting to update lists without touching files. 

I like the convenience of having it all in one command, but FWIW, this could be just one step with two commands in CI.

I think it'll simplify how we update lists to kibana. Right now, it looks like there is a single post command to "update rules" but I suspect pushing lists to the same route will not work (don't they have their own endpoints)? OR perhaps leave as-is and just have `RuleResource.import_rules` call both/3? endpoints.

Reply:
so this command is dependent on pre-formatting the rules appropriately. So _if_ exceptions are included, those overwrites simply allow control of whether to overwrite or not.

To directly manage lists independently, it is as you stated - there is a dedicated API, which would require at least 2 steps. If we choose to add that support, we should do it in a separate PR